cmd/openshift-install/create.go

```diff
@@ -268,13 +270,25 @@ func logComplete(directory string) error {
 	if err != nil {
 		return err
 	}
+	tfvarsFile := filepath.Join(absDir, cluster.TfVarsFileName)
+	f, err := os.Open(tfvarsFile)
+	if err != nil {
+		return err
+	}
+	defer f.Close()
+	tfVars, _ := ioutil.ReadAll(f)
+	var result map[string]interface{}
+	json.Unmarshal([]byte(tfVars), &result)
+	clusterName := result["cluster_name"]
+	baseDomain := result["base_domain"]
 	kubeconfig := filepath.Join(absDir, "auth", "kubeconfig")
 	pwFile := filepath.Join(absDir, "auth", "kubeadmin-password")
 	pw, err := ioutil.ReadFile(pwFile)
 	if err != nil {
 		return err
 	}
-	logrus.Infof("kubeadmin user password: %s", pw)
-	logrus.Infof("Install complete! The kubeconfig is located here: %s", kubeconfig)
+	logrus.Infof("Install complete! To manage the cluster with 'oc', the OpenShift CLI, first run 'export KUBECONFIG=%s'", kubeconfig)
+	logrus.Infof("The cluster is ready when this command succeeds (wait a few minutes): 'oc login -u kubeadmin -p %s'", pw)
```

Comment:
Should we provide an option to wait until this would work?  And turn it on by default, at least for the "no production clusters yet" phase?

Reply:
yes, but can't decide what's best way to do that? 
Need to wait for 2 things:
1) console route is ready
2) OAuth bits are ready (this happens as an update to the kubeapiserverconfig, essentially, AFAIK, so requires that things 'come back up')
To wait for login or to get the route, would require openshift-client, but I was trying to keep this simple. Thinking...